packages/timeline-state-resolver/src/integrations/httpSend/$schemas/options.json

```diff
@@ -71,6 +71,28 @@
 				"required": ["type", "url", "params"],
 				"additionalProperties": false
 			}
+		},
+		"oauthTokenHost": {
+			"type": "string",
+			"ui:title": "OAuth 2.0 Token Host",
```

Comment:
I think it would be useful to specify that this is the "base URL" of the oauth flow, i.e. the way the "token host" is described in the `simple-oauth2`: "Base URL used to obtain access tokens". I don't think "token host" is a widely used term in the OAuth world. Also, specifying that this is in fact a URL (and not, say a hostname + path) would be helpful.

Reply:
I made changes to the UI descriptions of this setting, and I added another field to allow specifying the path